contracts/CHANGELOG.md

```diff
@@ -1,8 +1,48 @@
 # @chainlink/contracts
 
+## 1.4.1
+
+### Minor Changes
+
+- [#164](https://github.com/smartcontractkit/chainlink-evm/pull/164) [`bf75f8a`](https://github.com/smartcontractkit/chainlink-evm/commit/bf75f8a) - Adding a variety of audited flavours of the CCT token (BurnMintERC20) that implements UUPS and Transparent Proxy upgradeability patterns. Brings proxy-related OZ contracts into vendor.
+
+- [#121](https://github.com/smartcontractkit/chainlink-evm/pull/121) [`e479a8d`](https://github.com/smartcontractkit/chainlink-evm/commit/e479a8d409) - Add BurnMintERC20WithDrip to replace the BurnMintERC677Helper. This should serve as the new standard BurnMintERC20 implementation.
```

Comment:
> standard BurnMintERC20 implementation

not `BurnMintERC20` but something like `drippable token` idk, it's just a token for testnet, we don't want customers accidentally deploying a token with a free mint function